src/global-styles.scss

```diff
@@ -1,14 +1,71 @@
-$example-color: cornflowerblue;
+// gray-700
+// $bg-color: #374151;
 
-.example-class {
-    color: $example-color;
+// zinc-900
+$bg-color: #18181b;
+$text-color: #f8fafc;
+
+// gray-800
+// $card-lv-1-color: #1f2937;
+
+// zinc-700
+$card-lv-1-color: #3f3f46;
+
+
+
+html {
+    background-color: $bg-color;
+    color: $text-color;
+    height: 100%;
+}
+
+body {
+    height: 100%;
+}
+
+.sveltekit-container {
+    height: 100%;
 }
 
 .button {
+    all: unset;
     padding: 8px;
     border-radius: 4px;
     border: 2px solid #999;
     text-decoration: none;
     cursor: pointer;
+    user-select: none;
     display: inline-block;
-}
\ No newline at end of file
+}
+
+a, a:visited, a:hover, a:active {
+    color: inherit;
+}
+
+// temporary class, pending addition of full size styling.
+// When you're on desktop viewport size, it will confine all UI to a mobile-sized area
+.mobile-sized-container {
+    background-color: $card-lv-1-color;
+    max-height: 600px;
+    max-width: 400px;
+    width: 100%;
+    height: 100%;
+    border-radius: 12px;
+    margin: auto;
+    padding: 20px;
+}
+
+input[type="text"] {
+    background-color: #1E1E1E;
+    border: 1px solid #333;
+    color: #E0E0E0;
+    padding: 10px 15px;
```

Comment:
Another unit to know for css sizing: `em`.  It does sizing relative to fontsizes.  It allows for responsive behavior when users change the browser's default font size.  i.e. for padding or margin, em allows for bigger fonts to have more space around them.

Reply:
Do you think we'd prefer `em` in general?

My only concern is that variable font size could make unintended things happen, like a flexbox item pushing to a new row when it's not supposed to.